pkg/chunkenc/pool.go

```diff
@@ -47,7 +47,7 @@ var (
 	// BufReaderPool is bufio.Reader pool
 	BufReaderPool = &BufioReaderPool{
 		pool: sync.Pool{
-			New: func() interface{} { return bufio.NewReader(nil) },
+			New: func() interface{} { return bufio.NewReaderSize(nil, 64*1024) },
```

Comment:
`64*1024` -> some `const`?

Reply:
It's only used here and so I think it read better like this.